lib/bundle/brew_installer.rb

```diff
@@ -90,6 +97,41 @@ def upgradable?
       BrewInstaller.formula_upgradable?(@name)
     end
 
+    def conflicts_with
+      @conflicts_with ||= begin
+        conflicts_with = Set.new
+        conflicts_with += @conflicts_with_arg if @conflicts_with_arg
+
+        if (formula_info = Bundle::BrewDumper.formula_info(@full_name))
+          if (formula_conflicts_with = formula_info[:conflicts_with])
+            conflicts_with += formula_conflicts_with
+          end
+        end
+
+        conflicts_with.to_a
+      end
+    end
+
+    def unlink_conflicts!
+      conflicts_with.each do |conflict|
+        if BrewInstaller.formula_installed?(conflict)
+          if ARGV.verbose?
+            puts <<-EOS
```

Comment:
It may be not available for test. But it's definitely available in bundle as it loads all of Homebrew core code. I think all you need to do is adding a dumpy `String#undent` method in `spec_helper.rb`.